scripts/codegen_driverwrappers/generate_driver_wrappers.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+import os
+import jinja2
+
+def render(tpl_path):
+    path, filename = os.path.split(tpl_path)
+    return jinja2.Environment(
+        loader=jinja2.FileSystemLoader(path or './')
+    ).get_template(filename).render()
```

Comment:
This would be more readable with intermediate variables. Something like:
```suggestion
def render(template_path: str) -> str:
        environment = jinja2.Environment(
        loader=jinja2.FileSystemLoader(os.path.dirname(template_path))
    )
    template = environment.get_template(os.path.basename(template_path))
    return template.render()
```